.eslintrc.json

```diff
@@ -19,6 +19,9 @@
         "plugin:@angular-eslint/recommended",
         "plugin:@angular-eslint/template/process-inline-templates"
       ],
+      "plugins": [
```

Comment:
I'm a little unclear on why this is now needed or why it wasn't needed before. `npm run lint` fails now without it.